let day = Day1()

do {
    let result = try day.run()
    print(result)
} catch {
    print(error)
}
